google/resource-snippets/pubsub-v1/alternatives/tf/pubsub.tf

```diff
@@ -1,64 +1,40 @@
 provider "google" {
-  project = "{PROJECT_ID}"
+  project = var.project_id
   region  = "{REGION}"
   zone    = "{ZONE}"
```

Comment:
Concrete values for these too? The goal is to allow anyone to try this out with minimal friction, like `git clone` this repo and doing `terraform init && terraform apply -var="deployment=my-deployment" -var="project_id=my_project"` after setting up auth. Setting sensible defaults here should work majority of the time.